sdk/python/feast/infra/offline_stores/bigquery.py

```diff
@@ -478,6 +490,43 @@ def persist(self, storage: SavedDatasetStorage):
     def metadata(self) -> Optional[RetrievalMetadata]:
         return self._metadata
 
+    def supports_remote_storage_export(self) -> bool:
+        return self._gcs_path
```

Comment:
does this actually return a `bool`? should this be `return self._gcs_path != None` or somethign?